### DISCUSSION TITLE
Complete the expression

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1094] [Galaxy Trivia] [answered]
> 📺 is not real 🧬. In real 🧬, ...

**Bonus:** Name the author.

**Comment by peterbaccaro** [comment]
**Television is not real life. In real life people actually have to leave the coffee shop and go to jobs.** is a quote from **Bill Gates**.

**Comment by thuongtruong1009** [comment, marked as answer]
Complete the expression: 📺 is not real 🧬. In real 🧬, "DNA is a complex molecule that contains the genetic instructions for the development and functioning of living organisms."

Author: Richard Dawkins
